libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/ContinueConversationLater.cs

```diff
@@ -32,7 +31,7 @@ public class ContinueConversationLater : Dialog
         /// The Kind name for this dialog.
         /// </summary>
         [JsonProperty("$kind")]
-        public const string Kind = "AzureQueues.ContinueConversationLater";
+        public const string Kind = "Microsoft.ContinueConversationLater";
```

Comment:
Not sure if we still do this, but I think we need to update test.schema with the new name (I don't know if test.schema gets regenerated on the build server but the copy in source control has the old name)

Reply:
Do you mean the schema in libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions? If so, that change is included in this PR. I have not found any other references in the source. Please let me know.

---
In reply to: [501018849](https://github.com/microsoft/botbuilder-dotnet/pull/4692#discussion_r501018849) [](ancestors = 501018849)

Reply:
I searched on my local copy of schema.tests and I see a reference to the old name, I do see it too on your branch [here](https://github.com/microsoft/botbuilder-dotnet/blob/moniriv/queueref/tests/tests.schema#L8), [here](https://github.com/microsoft/botbuilder-dotnet/blob/moniriv/queueref/tests/tests.schema#L453) and in a couple of other places.

It seems that tests.schema gets regenerated by the unit tests so I am not sure if this is relevant or not (if tests.schema is regenerated every time we may want to added to the gitignore, would this be the case @chrimc62?)